podaac/subsetter/subset.py

```diff
@@ -528,6 +528,17 @@ def compute_time_variable_name(dataset, lat_var):
     raise ValueError('Unable to determine time variable')
 
 
+def compute_utc_name(dataset):
+    """
+    Get the name of the utc variable if it is there to determine origine time
+    """
+    for var_name in list(dataset.data_vars.keys()):
+        if 'utc' in var_name.lower():
```

Comment:
this seems too broad; is there a way to be more specific here? This would match any variable where `utc` is found anywhere within it's name